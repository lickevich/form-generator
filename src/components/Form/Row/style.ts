import styled from "styled-components";

export const StyledRow = styled.div`
  padding: 20px;
`;
