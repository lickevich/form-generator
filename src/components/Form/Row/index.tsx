import React from "react";

import { StyledRow } from "./style";

const Row: React.FC<React.ReactNode> = ({ children }) => (
  <StyledRow>{children}</StyledRow>
);

export default Row;
