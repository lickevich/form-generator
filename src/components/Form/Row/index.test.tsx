import React from "react";
import { render } from "@testing-library/react";

import Row from "./";

describe("Row", () => {
  it("render the component", () => {
    const component = (
      <Row>
        <label htmlFor="123">Label</label>
        <input type="text" id="123" />
      </Row>
    );
    const { asFragment } = render(component);

    // @ts-ignore
    expect(asFragment(component)).toMatchSnapshot();
  });
});
