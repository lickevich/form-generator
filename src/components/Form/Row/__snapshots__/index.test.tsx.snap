// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdvvaa kTYFsi"
  >
    <label
      for="123"
    >
      Label
    </label>
    <input
      id="123"
      type="text"
    />
  </div>
</DocumentFragment>
`;
